### ISSUE TITLE
translate_columns dependencies

### ISSUE THREAD

**Issue by mattherick** [samlown/translate_columns#4]
Hi!

I want to add your translate_columns gem to my project. I get following error while bundling:

``` ruby
Bundler could not find compatible versions for gem "activerecord":
In Gemfile:
translate_columns (>= 0) ruby depends on
activerecord (~> 3.0.0) ruby
rails (= 3.2.12) ruby depends on
activerecord (3.2.12)
```

Is it possible, that you update your gem to be compatible with rails 3.2?

**Comment by samlown** [comment]
Hi. Its been a seriously long time since I've used this gem to be honest!

If you have time, I'd suggest cloning the project and sending me a pull request with the changes to support newer versions of ActiveRecord.

**Comment by mattherick** [comment]
closed because of #5 update.
